app/controllers/trainees/degrees_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
 
     def edit
       authorize trainee
-      @degree = trainee.degrees.find(params[:id])
+      @degree = trainee.degrees.find_by(slug: params[:id])
```

Comment:
Can all these calls to `find_by` be our sluggable `from_param`?

Reply:
yup and done! https://github.com/DFE-Digital/register-trainee-teachers/pull/411/commits/ec826dba2e4813cf46436aec0706d4ea85b26527